utils/crypto/keychain/ledger/ledger_keychain.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package ledger
```

Comment:
the name of the package is `ledger`, so the names of the exported functions here should not have `ledger` in them, otherwise it stutters. 

For example: "`ledger.NewKeyChain`